pkg/gameservers/sdkserver_test.go

```diff
@@ -157,7 +171,7 @@ func TestSidecarHealthLastUpdated(t *testing.T) {
 
 	sc, err := defaultSidecar(m)
 	assert.Nil(t, err)
-	sc.healthDisabled = false
+	sc.health = v1alpha1.Health{Disabled: true}
```

Comment:
Doesn't it change the intent of the test?

Reply:
Fixed! Incoming shortly.